wpilibj/src/main/java/edu/wpi/first/wpilibj/Notifier.java

```diff
@@ -18,17 +18,17 @@
   // The C pointer to the notifier object. We don't use it directly, it is
   // just passed to the JNI bindings.
   private final AtomicInteger m_notifier = new AtomicInteger();
-  // The time, in microseconds, at which the corresponding handler should be
-  // called. Has the same zero as Utility.getFPGATime().
-  private double m_expirationTime;
+  // The time, in seconds, at which the corresponding handler should be
```

Comment:
Once again, it seems like something is fishy here. Was the code updated without the comments a while ago, or what?

Reply:
The comment used to be wrong. This variable is seconds. You can see how we multiply by 1e6 when calling updateAlarm() because that function takes microseconds as an integer.

Reply:
Utility.getFPGATime() Used to return a long in microseconds. When the code was updated, the comment wasn't updated. As calcmogul stated the code all looks correct.